### PR TITLE
66 terminal output pipeline

### PR DESCRIPTION
**Pull request by aryanbattula**
created the terminal output pipeline

### CODE REVIEW

File: EosGround/database/pipeline/pipelines/terminal_output_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    def extract(self, session: Session) -> Query:
+
+        return session.query(ReceivedPackets).filter_by(packet_type=Type.CUTDOWN, processed=False)\
+            .order_by(ReceivedPackets.id)
+
+    def transform(self, session: Session, record: namedtuple):
+        print(f"transforming terminal_output_pipeline row id={record.id}")
+        received_packet_id = record.id
+        packet_body = CutDown.decode(record.packet_body)
```
Comment (VanL15) [inline EosGround/database/pipeline/pipelines/terminal_output_pipeline.py:33]: Same here with `CutDown.decode`.

**Review by VanL15 — CHANGES_REQUESTED** [review]
See comments, only a few things to change. Also, if you disagree with any of the changes let me know.

File: EosGround/database/models/eos/transmit.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    __tablename__ = 'transmit_table'
+    __table_args__ = {'schema': SCHEMA}
+
+    id: Mapped[int] = mapped_column(Integer, Identity(start=1), primary_key=True, init=False)
+    time_sent: Mapped[datetime] = mapped_column(TIMESTAMP)
+    packet_type: Mapped[int] = mapped_column()
+    sender: Mapped[int] = mapped_column()
+    priority: Mapped[int] = mapped_column()
+    destination: Mapped[int] = mapped_column()
+    generate_time: Mapped[int] = mapped_column()
```
[the remark this answers is not in the document]
Comment (aryanbattula) [reply EosGround/database/models/eos/transmit.py:21]: good catch

**Review by IonLi09 — APPROVED** [review]
pipeline was tested on my laptop. lgtm!

**Pull request merged**